synapse/tests/test_tools_apikey.py

```diff
@@ -0,0 +1,227 @@
+import datetime
+
+import synapse.exc as s_exc
+import synapse.common as s_common
+
+import synapse.lib.time as s_time
+import synapse.lib.output as s_output
+
+import synapse.tests.utils as s_test
+import synapse.tools.apikey as s_t_apikey
+
+async def getApiKeyByName(core, name):
+    keys = {k.get('name'): k async for k in core.getApiKeys()}
+    return keys.get(name)
+
+class ApiKeyTest(s_test.SynTest):
+
+    async def test_tools_apikey(self):
+        async with self.getTestCore() as core:
+
+            await core.auth.addUser('blackout')
+
+            rooturl = core.getLocalUrl()
+            blckurl = core.getLocalUrl(user='blackout')
+
+            # Add API keys
+            argv = (
+                '--svcurl', rooturl,
+                'add',
+                'rootkey00',
+                '-d', '120',
+            )
+            outp = s_output.OutPutStr()
+            self.eq(0, await s_t_apikey.main(argv, outp=outp))
+
+            self.isin('Successfully added API key with name=rootkey00.', str(outp))
+            rootkey00 = await getApiKeyByName(core, 'rootkey00')
+
+            self.isin(f'Iden: {rootkey00.get("iden")}', str(outp))
+            self.isin('  API Key: ', str(outp))
+            self.isin('  Name: rootkey00', str(outp))
+            self.isin(f'  Created: {s_time.fmttime(rootkey00.get("created"))}', str(outp))
```

Comment:
Is there a reason not to use `s_time.repr()` ?

Reply:
Because I didn't know about it.